documentation/release-notes/4.3.1.md

```diff
@@ -0,0 +1,263 @@
+<!--
+    Copyright 2023 Loïc Lamarque.
+    Use of this source code is governed by the MIT license.
+-->
+
+# 4.3.1
+
+_Release date: 2023-09-25 | [Full changelog]._
+
+This patch release includes new experimental type converters suffixed by
+`OrNull` and `OrThrow`, source compatibility improvements with [Kotlin] and
+documentation updates.
+
+[full changelog]: https://github.com/kotools/types/blob/main/CHANGELOG.md#431
+[kotlin]: https://kotlinlang.org
+
+## Installation
+
+Add the following dependency to your project for using Kotools Types 4.3.1.
+
+<details open>
+<summary>Gradle - Kotlin DSL</summary>
+
+```kotlin
+implementation("org.kotools:types:4.3.1")
+```
+</details>
+
+<details>
+<summary>Gradle - Groovy DSL</summary>
+
+```groovy
+implementation "org.kotools:types:4.3.1"
+```
+</details>
+
+<details>
+<summary>Maven</summary>
+
+```xml
+<dependencies>
+    <dependency>
+        <groupId>org.kotools</groupId>
+        <artifactId>types</artifactId>
+        <version>4.3.1</version>
+    </dependency>
+</dependencies>
+```
+</details>
+
+Visit the [API reference] for more details on the declarations provided by this
+library.
+
+[api reference]: https://types.kotools.org
```

Comment:
The installation guide should also include projects using specific platforms (Kotlin/JVM, Kotlin/JS and Kotlin Native).